FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/mock/MockRobot.java

```diff
@@ -41,7 +52,7 @@ public void setLidarSpeed(double speed) {
 	}
 
 	public SensorData getSensorData() {
-		// TODO Auto-generated method stub
+		// Should this return something else?

```

Comment:
Because currently `SensorData` isn't in an interface, and we didn't want to include all the wrapping and making a mock in this PR.